lib/matplotlib/projections/polar.py

```diff
@@ -1202,9 +1202,13 @@ def set_ylim(self, bottom=None, top=None, emit=True, auto=False,
                                  'argument and kwarg "ymax"')
             else:
                 top = ymax
-        if top is None and len(bottom) == 2:
-            top = bottom[1]
-            bottom = bottom[0]
+        if top is None:
```

Comment:
`set_xlim` uses

~~~
        if right is None and np.iterable(left):
            left, right = left
~~~

I think it's reasonable to use the same pattern here.

Note: This will explicitly fail `set_rlim((1, 2, 3))`. Your code would let that pass, and I assume it would have failed later on, since the assumption from here on is that bottom is a scalar.

Reply:
Done!